.travis.yml

```diff
@@ -28,10 +28,6 @@ before_script:
   - if [ $TRAVIS_RUST_VERSION = $CLIPPY_TOOLCHAIN ]; then rustup component add clippy-preview --toolchain=$CLIPPY_TOOLCHAIN; fi
   - curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh -s -- -f
 
-script: ./test.sh
 
 script:
   - yarn
-  - yarn build:all
```

Comment:
I don't think it is ok for a PR to stop building the project and remove all tests.